### ISSUE TITLE
bug: Bug in live chat, when YT RXV is connected to TV

### ISSUE THREAD

**Issue by dark-caster** [inotia00/revanced-patches#34]
### Type

Other

### Bug description

When YT RXV is connected to a TV, when opening a LIVE video and **clicking "live chat", the android keyboard keeps closing by itself!** On some devices, the keyboard remains open to write the message (however, when you click on search for emojis to add emojis to the message, the keyboard closes by itself).

This bug only occurs on LIVE videos and when the YT RXV app is connected to the TV. In other videos that are not live" it is possible to write in the comments normally, that the keyboard does not close by itself.
     If the YT RXV is not connected to the TV, the bug does not happen, it is possible to write in the live chat normally and the keyboard does not close by itself.

I tested it on two phones, on a Moto G30 (with Android 12) and on a Moto G5 (with Android 8.1.0).

**On the Moto G5**: the keyboard does not open at all, it is only possible to send messages, writing them in another application and then copy and paste the messages in the live chat.

**On the Moto G30**: the keyboard opens and it is possible to write messages, but when you click on the emoji button on the android keyboard and then go to the field to search for emojis, the keyboard closes by itself.
      When using the Google translator function built into the android keyboard, to send already translated messages without having to open Google Translate, the keyboard also closes by itself.

I'm using YT RXV 17.41.36 (no root), on both devices (they don't have root).

### Steps to reproduce

1. Connect the YT RXV to the TV (via the "cast" button from the app itself);

2. Put any LIVE video to play on TV;

3. Open the "live chat" and click on the field to write a message.

### Relevant log output

```shell
I don't know how to get the log on YT RXV.
```

### Screenshots or videos

https://user-images.githubusercontent.com/113958057/198120952-749f72fe-8c21-4092-b096-f8ad4c26c7c4.mp4

https://user-images.githubusercontent.com/113958057/198128296-a2baf5a1-28c1-4fd2-8071-a38bb2cfa195.mp4

### Additional context

Moto G5 (Android 8.1.0)
Architecture: armeabi-v7a
No root

Moto G30 (Android 12)
Architecture: arm64-v8a
No root

YT RXV 17.41.36 (no Root)
Dark theme (not black)
Architecture: armeabi-v7a

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by inotia00** [comment]
1. Connect to TV via cast button
2. Play LIVE video on TV
3. Open live chat and enter chat

Issue does not reproduce

tested environment
- YouTube v17.41.37 (recommended version)
- revanced-patches v2.85.2
- revanced-integrations v0.55.0

After setting the same as my test environment, check if the issue is reproduced again.
If the issue still recurs, please reopen the issue.

**Comment by dark-caster** [comment]
Good Morning!

I just installed YouTube Revanced Extended which is available there in the Telegram group (Extended Lite), the version: 17.42.35.

I noticed that the same problem continues. The keyboard keeps closing by itself, this time I tested it on 3 cell phones.

On Moto G30 (which has arm64-v8a architecture, the problem does not occur.

I tested it on two cell phones that have the architecture armeabi-v7a and that's where the problem occurs. I tested it on: Moto G5 (Android 8.1.0) and Moto G7 Play (Android 10).
